website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -33,6 +33,7 @@ the traffic redirection command is automatically set up via an init container.
 * To use transparent proxy on Kubernetes, Consul-helm >= `0.32.0` and Consul-k8s >= `0.26.0` are required in addition to Consul >= `1.10.0`.
 * If the default policy for ACLs is "deny", then Service Intentions should be set up to allow intended services to connect to each other.
 Otherwise, all Connect services can talk to all other services.
+* Linux worker nodes within a Kubernetes cluster should have `iptables` installed. 
```

Comment:
```suggestion
* If using Transparent Proxy, all worker nodes within a Kubernetes cluster must have the `ip_tables` kernel module running, e.g. `modprobe ip_tables`. 
```

Reply:
Bonus points if we can find a public document which shows someone how to install it and add it into /etc/modules.rc (or whatever the modern equivalent is)

Reply:
I'll leave this as is, I believe most Linux distros have this kernel module enabled but the installation is likely specific to a distro.